.github/workflows/reusable-add-reviewers-to-pr.yml

```diff
@@ -0,0 +1,52 @@
+name: Add reviewers to PRs
+
+on:
+  pull_request_target:
+    branches: ['master', 'next']
+    types: ['labeled']
+  workflow_call:
+    inputs:
+      team-slug:
+        description: 'The slug of the team from which reviewers get collected'
+        required: true
+        type: string
+      label-name:
+        description: 'The name of the label that triggers the action'
+        required: true
+        type: string
+    secrets:
+      token:
+        description: 'The github token to use for the API calls. You can use the GITHUB_TOKEN secret'
+        required: true
+        type: string
+
+permissions: {}
+
+jobs:
+  add-reviewers-to-pr:
+    # Tests that label is added on the PR
+    if: ${{ github.event.label.name == inputs.label-name }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - id: get-members
+        run: |
+          DATA=$(gh api \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            /orgs/${{ github.repository_owner }}/teams/${{ inputs.team-slug }}/members?role=maintainer&per_page=100 \
```

Comment:
I don't understand this logic. Is this PR meant to experiment with reusable workflow?